api/src/db/models.rs

```diff
@@ -565,6 +565,12 @@ pub struct GithubRepository {
   pub created_at: DateTime<Utc>,
 }
 
+impl GithubRepository {
+  pub fn get_head_url(&self) -> String {
+    format!("https://github.com/{}/{}/blob/HEAD", self.owner, self.name)
```

Comment:
I feel like images should link to raw.githubusercontent.com

Reply:
@lucacasonato adressed